package.json

```diff
@@ -228,6 +228,11 @@
           "default": false,
           "description": "Whether to run code in Integrated Terminal."
         },
+        "code-runner.terminalRoot": {
+          "type": "string",
+          "default": "",
+          "description": "Replace the drive letter X: with `${terminalRoot}x/` for custom terminals."
```

Comment:
The wording is hard for user to understand:

- You need to mention it is for Windows user to use Linux bash
- You'd better to add a simple example here: e.g. `/cygdrive/` for cygdrive, `/` for Git Bash